lib/storage.js

```diff
@@ -286,6 +286,10 @@ api.insert = async ({edvId, doc}) => {
   if(doc.sequence < 0) {
     throw new TypeError('"doc.sequence" must be a non-negative integer.');
   }
+  if(!Number.isSafeInteger(doc.sequence) ||
```

Comment:
looks like there's an opportunity to DRY this up a bit.  How about a private helper `_validateDocSequence(..)` helper used in both places?

Also, technically the value has not exceeded MAX_SAFE_INTEGER so I wonder if that's going to be confusing. Not sure what to say here that lets people know what the safe values are unless its exacly `exceeded MAX_SAFE_INTEGER - 1`.  @mandyvenables @dlongley thoughts, suggestions?

Reply:
In my earlier comment I suggested just saying the sequence number is too large.

Reply:
Moved the check to a private helper function and updated the error message to '"doc.sequence" number is too large.'